.github/workflows/release-api-gateway-2.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
     uses: ./.github/workflows/call-unit-lint.yaml
 
   integration-tests:
-    needs: build-image
+    needs: [build-image, check-prerequisites]
```

Comment:
As build-image already requires check-prerequisites we don't need to add it in here

Reply:
OK, It's needed for the needs below 🆗